pkg/kubelet/kubelet.go

```diff
@@ -1275,7 +1270,6 @@ func (kl *Kubelet) initializeModules() error {
 func (kl *Kubelet) initializeRuntimeDependentModules() {
 	if err := kl.cadvisor.Start(); err != nil {
 		// Fail kubelet and rely on the babysitter to retry starting kubelet.
-		// TODO(random-liu): Add backoff logic in the babysitter
```

Comment:
note: no action needed in kubelet, process monitor is not a concept in k/k repositories.